cli/src/semgrep/commands/ci.py

```diff
@@ -219,7 +219,7 @@ def ci(
         output_format=output_format,
     )
 
-    state.metrics.configure(metrics, metrics_legacy)
+    state.metrics.configure(metrics, metrics)
```

Comment:
Why are we sending the argument twice?

Reply:
oops this was left over from a previous change